uber/tests/models/test_badge_funcs.py

```diff
@@ -206,6 +310,80 @@ def test_non_preassigned_deletion(self, session):
         session.delete(session.query(Attendee).filter_by(badge_type=c.ATTENDEE_BADGE).first())
         session.commit()
 
-# TODO: more unit tests for changes/deletions after custom badges have been ordered
```

Comment:
Basking needs some kind of mascot.  Maybe a "baskalisk", though unfortunately all of the Google searches I did for "baskalisk" didn't return different results than for the correct spelling.  Alas.